docs/INSTALL.md

```diff
@@ -53,36 +113,6 @@ sudo dnf install -y \
 brew install cmake qt@5
 ```
 
-### Getting sources
-
-Clone the repository:
-
-```bash
-git clone --recursive https://github.com/organicmaps/organicmaps.git
-```
-
-Default clone destination directory is organicmaps.
-
-Update git submodules (sometimes doesn't work automatically):
-
-```bash
-git submodule update --init --recursive
-```
-
-Configure the repository as opensource build:
-
-```bash
-./configure.sh
-```
-
-or with private repository
-
-```bash
-./configure.sh <private-repo-name>
-```
-
-You can check usage `./configure.sh --help`
-
 ### Building
 
 With all tools installed, just run `tools/unix/build_omim.sh`.
```

Comment:
Also adding default `desktop` target can help people.

Reply:
Thanks! I'll do it in next PR, my idea was to limit the current PR to deduplication and update of the Getting sources section.